python/ray/train/v2/_internal/callbacks/state_manager.py

```diff
@@ -109,8 +109,6 @@ def after_controller_state_update(
                 run_id=self._run_id,
             )
 
-        # TODO: ABORT is not handled by Controller.
-
```

Comment:
Were you thinking of adding an `AbortedState`? @matthewdeng

Reply:
added